Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/README.md

```diff
@@ -0,0 +1,3 @@
+# Example1 - Background color changer
```

Comment:
Use normal English. Give proper grammar. Space after Example.

Reply:
Required changes made in latest commit.